doc/guides/writing-tests.md

```diff
@@ -18,6 +18,13 @@ Add tests when:
 - Fixing regressions and bugs.
 - Expanding test coverage.
 
+## Where does the test go
+One can refer [test directory structure](https://github.com/nodejs/node/tree/master/test)
+to decide where to put your test cases. If you want to find any current test that exists then
+go to the defined [test directory structure](https://github.com/nodejs/node/tree/master/test) 
+and search for similar APIs and see if there is similar file available. 
```

Comment:
Overall, this is a bit too verbose and repeats the same information twice. We also do our best to avoid using "you", "your", "one" and similar. We could communicate the same thought in a shorter sentence:

> See the [test directory structure overview](https://github.com/nodejs/node/blob/master/test/README.md) for an outline of existing test types and their locations.

That could probably be improved further. (Note that the link above goes directly to the README rather than the test folder.)

Reply:
Does above description answers nodejs#18774 second issue?

Reply:
It probably doesn't but I don't think the current one does either. I would go with something like:

> When deciding on whether to expand an existing test file or create a new one, consider going through the files related to the subsystem (e.g. ones starting with "test-streams" for `lib/streams.js`) and assessing whether a related test already exists.

It's possible that should be even more detailed, like for example explaining the difference between parallel and sequential, but that could get quick tricky and complicated.

Reply:
How about 
> See the test [directory structure overview](https://github.com/nodejs/node/blob/master/test/README.md)
for an outline of existing test types and their locations.
For finding test that exist, look for similar APIs or file in directory.

this ?

Reply:
Thanks making changes with the above one. :)

Reply:
changes made.

Reply:
Thank you for feedback.